index.html

```diff
@@ -6,7 +6,7 @@
           <link rel="stylesheet" href="styles/main.css">
           <link rel="stylesheet" href="styles/mob.css">
           <link rel="stylesheet" href="styles/baguetteBox.css">
-          <link rel="stylesheet" href="styles/baguetteBox.js">
+          <script src="styles/baguetteBox.js"></script>
```

Comment:
Don't import scripts using the `<link>` element.  Use the `<script>` element instead